kitsune/wiki/tests/test_views.py

```diff
@@ -23,12 +23,327 @@
     RedirectRevisionFactory,
     RevisionFactory,
     TemplateDocumentFactory,
+    TestCaseBase,
+    TranslatedRevisionFactory,
     new_document_data,
 )
 from kitsune.wiki.views import _document_lock_check, _document_lock_clear, _document_lock_steal
 
 
-class RedirectTests(TestCase):
+class VisibilityTestsBase(TestCaseBase):
```

Comment:
With the tests let's be very repetitive and verbose so it can be very obvious what the testis about. I know that it will be add bolierplate code but this will make each test much more atomic and self contained